app/views/pages/home.html.erb

```diff
@@ -1,11 +1,12 @@
 <section class="demo">
+  <% @lexer = Rouge::Lexer.all.sample %>
```

Comment:
If you want to rip out high voltage, I'd be happy to move it to a controller - it's all a bit magic for me.